test/Microsoft.AspNet.Mvc.Xml.Test/project.json

```diff
@@ -0,0 +1,19 @@
+﻿{
+  "compilationOptions": {
+    "warningsAsErrors": "true"
+  },
+  "dependencies": {
+    "Microsoft.AspNet.Mvc" : "6.0.0-*",
+    "Microsoft.AspNet.Mvc.Xml" : "6.0.0-*",
```

Comment:
No, there are a lot of tests in Core that depend on the `Microsoft.AspNet.Mvc.Xml` package. So that needs to stay.